include/maddy/parser.h

```diff
@@ -131,6 +134,8 @@ class Parser
     this->strikeThroughParser->Parse(line);
 
     this->inlineCodeParser->Parse(line);
+
+    this->italicParser->Parse(line);
```

Comment:
In `ItalicParser.h` the comment says `Has to be used before the EmphasizedParser` - but here it isn't?

Reply:
ctrl-c, ctrl-v mistakes.